pyserini/dsearch/__main__.py

```diff
@@ -28,6 +28,8 @@
                     help="Path to Faiss index or name of prebuilt index.")
 parser.add_argument('--topics', type=str, metavar='topic_name', required=True,
                     help="Name of topics. Available: msmarco_passage_dev_subset.")
+parser.add_argument('--query_emb', type=str, metavar='path to query embedding or query name', required=True,
```

Comment:
later on, you have `encoded_query_info.py` and `load_encoded_queries`, so the option should be `--encoded-queries` to be consistent? Also, dash instead of underscore to make it look prettier?